examples/full-example/app.js

```diff
@@ -47,4 +47,27 @@ function bundle(req, bundle, variations) {
     return '<script src="'+url+'"></script>';
 }
 
+function entryMap(req, bundle, variations) {
+    // req.mendel.getBundleEntries contains all bundles as keys and array of
+    // entries that were used and normalized by variations, this allows apps
+    // to do their specific logic with their bundles
+    var bundles = req.mendel.getBundleEntries();
```

Comment:
could move this line out of this function, so we dont have to call x times for x number of lazy bundles